deku/node/networking.re

```diff
@@ -178,7 +178,7 @@ module Withdraw_proof = {
 module Ticket_balance = {
   [@deriving yojson]
   type request = {
-    address: Address.t,
+    address: Address.Implicit.t,
     ticket: Ticket.t,
   };
   [@deriving yojson]
```

Comment:
This is using `Address.Implicit.t` because I don't know if/how it should work for contracts.

Reply:
we probably should have balance on contracts to comply with tezos, as people can send tez directly to contract